tests/test_main.py

```diff
@@ -265,9 +267,18 @@ def test_conda_run_conda_exe():
         "python",
         "-c",
         "import sys,os;print(os.environ['CONDA_EXE'])",
+        *(("--log-file", str(log_file)) if with_log else ()),
         check=True,
         text=True,
         capture_output=True,
         env=env,
     )
     assert os.path.realpath(process.stdout.strip()) == os.path.realpath(CONDA_EXE)
+    if with_log:
+        assert log_file.exists()
+        log_text = log_file.read_text().strip()
+        if sys.platform.startswith("win") and os.environ.get("CI"):
+            # on CI, setup-miniconda registers `test` as auto-activate for every CMD
+            # which adds some unnecessary stderr output; so, only read the first line
+            log_text = log_text.split("\n")[0]
+        assert os.path.realpath(log_text.strip()) == os.path.realpath(CONDA_EXE)
```

Comment:
Do we want to assert whether the process had any output? All of it should have technically gone to the logfile, right?

```suggestion
        assert os.path.realpath(log_text.strip()) == os.path.realpath(CONDA_EXE)
        assert not process.stdout + process.stderr
```